opm/parser/eclipse/EclipseState/Edit/EDITNNC.hpp

```diff
@@ -0,0 +1,48 @@
+/*
+  Copyright 2018 Equinor AS
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#ifndef OPM_COMMON_EDITNNC_HPP
+#define OPM_COMMON_EDITNNC_HPP
+
+#include <opm/parser/eclipse/EclipseState/Grid/NNC.hpp>
+namespace Opm
+{
+
+/// Represents edit information for non-neighboring connections (NNCs, faults, etc.)
+class EDITNNC
+{
+public:
+    EDITNNC() = default;
+
+    /// Construct from input deck
+    explicit EDITNNC(const Deck& deck);
+    /// \brief Get an ordered set of EDITNNC
+    const std::vector<NNCdata>& data() const
+    {
+        return m_editnnc;
+    }
+    /// \brief Get the number of entries
+    size_t size() const;
+    /// \brief Whether EDITNNC was empty.
```

Comment:
I know opinions go in all directions on this - but personally I consider comments of the type: `Get the number of entries` on a method called `size()` to be just noise.

Reply:
I would add `begin()` and `end() ` methods to thus class.

Reply:
I took NNC as a template for this class. That one must missing that, too. I am not opposed to adding them but maybe we should do this consistently for both classes in a separate PR.